docs/design/openstack/bootstrap-ignition-shim-asset.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Bootstrap Ignition Shim Asset
+authors:
+  - "@iamemilio"
+reviewers:
+  - TBD
+  - "@abhinavdahiya"
+  - "@wking"
+approvers:
+  - TBD
+  - "@abhinavdahiya"
+  - "@wking"
+creation-date: 2019-09-04
+last-updated: 2019-09-04
+status: implementable
+---
+
+# Bootstrap Ignition Shim Asset
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+
+## Summary
+
+There are a few platforms that use an Ignition shim for their bootstrap machine.
+Rather than generate this shim in Terraform, platforms should be able to **Opt-In**
+to generate a bootstrap Ignition shim in a Go based installer asset. The shim serves
+as a way to avoid hitting metadata size limits while still providing the machine with
+the data it needs to succesfully fetch the main Ignition config.
+
+## Motivation
+
+The motivation for this is largely due to the sluggish pace of updates to the
+Terraform Ignition Provider, which currently only supports ignition v2.1.0.
```

Comment:
wouldn't be better to spend the effort in taking the ownership of that provider and improve it ? initially the effort was started by the CoreOS folks and to me it would make more sense to continue the work there as more consumers will leverage the work on the tf ignition provider: FCoS/ UPIs OCP templates/ even typhoon project etc

Reply:
We have opened an issue there, and tried to stir up development. Updating the Terraform Ignition Provider ourselves seems a little out of scope for our team though (OpenShift on OpenStack). We are not keen on mounting a development effort in a project we have no experience in if we don't have to.